scripts/lib/CIME/test_scheduler.py

```diff
@@ -218,13 +218,15 @@ def __init__(self, test_names, test_data=None,
                         else:
                             self._update_test_status(test, phase, TEST_PEND_STATUS)
                             self._update_test_status(test, phase, status)
+                logger.info("Using existing test directory %s"%(self._get_test_dir(test)))
```

Comment:
Since this log output is inside the self._tests loop, wouldn't you end up with identical 'Using existing test directory' log entries if there is more than one test? I suggest either adding the test name to the log output (see below) or move the log outside the for loop.
`"Using existing test directory %s for "%(self._get_test_dir(test), test)`